e2e/tests/smoke/block_storage_smoke_test.go

```diff
@@ -40,9 +40,9 @@ func (suite *BlockStorageTestSuite) SetupTest() {
 
 	require.Nil(suite.T(), err)
 
-	rookInfra.ValidateAndSetupTestPlatform()
+	rookInfra.ValidateAndSetupTestPlatform(env.InstallRook == "true")
```

Comment:
could the `env.InstallRook` be of type `bool`? this doesn't seem like the right place to parse it